core/src/storage/impls/storage_db/kvdb_sqlite.rs

```diff
@@ -178,10 +183,7 @@ impl<ValueType> KvdbSqlite<ValueType> {
 
         // Table for string key.
         connection
-            .execute(
-                &statements.create_table,
-                &[&&bytes_key_table_name as SqlBindableRef],
-            )?
+            .execute(&statements.create_table, &[] as &[SqlBindableRef])?
```

Comment:
Here we should create a statements object, under the name of "bytes_key_table_statements".

We should remove create_table_number_key from the statements because we now always have two set of statements.